Doc/library/smtplib.rst

```diff
@@ -510,12 +510,17 @@ An :class:`SMTP` instance has the following methods:
    specified in :rfc:`5322`\: *from_addr* is set to the :mailheader:`Sender`
    field if it is present, and otherwise to the :mailheader:`From` field.
    *to_addrs* combines the values (if any) of the :mailheader:`To`,
-   :mailheader:`Cc`, and :mailheader:`Bcc` fields from *msg*.  If exactly one
-   set of :mailheader:`Resent-*` headers appear in the message, the regular
-   headers are ignored and the :mailheader:`Resent-*` headers are used instead.
-   If the message contains more than one set of :mailheader:`Resent-*` headers,
-   a :exc:`ValueError` is raised, since there is no way to unambiguously detect
-   the most recent set of :mailheader:`Resent-` headers.
+   :mailheader:`Cc`, and :mailheader:`Bcc` fields from *msg*.  If there's no
+   :mailheader:`Date` header inside the message, ``send_message`` will add one to the data.
+   If exactly one set of :mailheader:`Resent-*` headers appear in the message,
+   the regular headers are ignored and the :mailheader:`Resent-*` headers are
+   used instead.  If the message contains more than one set of
+   :mailheader:`Resent-*` headers, a :exc:`ValueError` is raised, since there
+   is no way to unambiguously detect the most recent set of
+   :mailheader:`Resent-` headers.
+
+   .. versionchanged:: 3.2
+      Support to add :mailheader:`Date` header to the message if one does not exist.
```

Comment:
I don't think we're gonna backport that to 3.2 :P

Should it be 3.14?

Reply:
thanks! i've changed it to 3.15 as I feel it's unlikely this will land for 3.14, but can change it down to 3.14 if there's a lot of support before the cutoff :p

Reply:
Just going jn should already be just good 😆